docs/docs/services/licenses.md

```diff
@@ -35,28 +38,39 @@ support multiple instances if needed.
 
 ### Health check
 
-For health checking, the service exposes `/v1/health` on port 9200, and it always responds with `200 OK`.
+For health checking, the service exposes `/health` on port 9200, and it always responds with:
+
+``` json
+{
+"alive": true
+}
+```
 
 ### Metrics
 
-For Prometheus metrics scraping, the service exposes `/v1/metrics` on port 9200. It provides the following
-metrics:
+For Prometheus metrics scraping, the service exposes `/metrics` on port 9200. It provides the following
+metrics associated with your license:
 
 | Name | Type | Description |
 | ---- | ---- | ----------- |
-| http_requests_total | COUNTER | Total number of HTTP requests since service start.<br>It includes both successful and unsuccessful requests.<br>The HTTP method and endpoint path are provided as labels. |
-| http_request_failures_total | COUNTER | Total number of HTTP request failures since service start.<br>The HTTP method and endpoint path are provided as labels |
-| http_request_duration_seconds | HISTOGRAM | Durations of HTTP request since service start.<br>The HTTP method and endpoint path are provided as labels. |
-| active_instances | GAUGE | Number of active instances (engines with an active license). |
-| license_expiry_seconds | GAUGE | Number of seconds until the license expires. |
+| license_time_consumption | GAUGE | Number of license minutes consumed this month. This metrics is only showed if any minutes have been consumed. |
+| license_time_total | GAUGE | The total amount of license minutes per month your license gives you. |
+
+To see an example of how you can use these metrics to create dashboards of your license consumption,
+see the [core-using-licenses](https://github.com/qlik-oss/core-using-licenses) repository.
 
 ### Logging
 
 This service complies with the [Logging](../conventions/logging.md) conventions.
 By default, the minimum log level is `info`.
-However, for Microsoft libraries, the default minimum logging level is set to `warning` to avoid too verbose logging.
 
-You can override the minimum logging level by providing the `LOG_LEVEL` environment variable.
-Note that if you provide this variable it will affect the logging level for all components in the License Service,
-including Microsoft libraries. We recommend that you use the default minimum level in a production environment,
-and only override the level for debugging purposes.
+You can override the minimum logging level by providing the `LICENSES_LOG_LEVEL` environment variable.
+
+### License Events
+
+If no valid license exists or all license minutes are already consumed and the Qlik Associative Engine
```

Comment:
This section is a little bit weird, it's not part of the licenses _service_ (which this page primarily is about), not sure where to fit it though, perhaps a licensing _tutorial_  accompanying the new git repo?

Reply:
Hmm, maybe. But as a user, I would not want the info spread out too much. And what I am interested in is how to run it and what to do if (and why) I get errors.